### ISSUE TITLE
Gdx.audio.getAvailableOutputDevices() is not updated if devices are added or removed

### ISSUE THREAD

**Issue by TCreutzenberg** [libgdx/libgdx#7148]
@Berstanio @actions-user : Thank you very much for adding Gdx.audio.getAvailableOutputDevices()!
I found an issue with it:

#### Issue details
Gdx.audio.getAvailableOutputDevices() does always return the same list of audio devices which are the devices connected when the application started. The list is not updated if devices are added or removed.

#### Reproduction steps/code
String[] x = Gdx.audio.getAvailableOutputDevices();
Add or remove a new audio device
String[] y = Gdx.audio.getAvailableOutputDevices();
x equals y

#### Version of libGDX and/or relevant dependencies
1.11.1-SNAPSHOT

#### Please select the affected platforms
Desktop

**Side note:**
(To create less garbage, it might be a good idea to maybe replace
private String[] lastAvailableDevices
with a libGDX Array<T> so you could remove the creation of a new ArrayList in the new thread: "List<String> currentDevicesList = new ArrayList<>(Arrays.asList(currentDevices));")

**Comment by Berstanio** [comment]
What OS do you use and what OS version? Does the OS recognize the removed/added audio device? Does the output device swapping works, if the output device gets disconnected?

> (To create less garbage, it might be a good idea to maybe replace
private String[] lastAvailableDevices
with a libGDX Array so you could remove the creation of a new ArrayList in the new thread: "List currentDevicesList = new ArrayList<>(Arrays.asList(currentDevices));")

The observer thread runs one time every second, I think the additional object is neglectable.

**Comment by TCreutzenberg** [comment]
It's Windows 10 (with latest updates). The OS

**Comment by TCreutzenberg** [comment]
If I set to auto-select with Gdx.audio.switchOutputDevice(null); and I plug in my headphones, the output changes to the headphones. But the list retrieved with Gdx.audio.getAvailableOutputDevices() does not contain the headphones.

**Comment by HydrolienF** [comment]
`Gdx.audio.getAvailableOutputDevices()` is working as expected for me on master (6db0232e4cc7a4d1a2960c2db2ece406c5c350a9)
I have triied to add a new blutooth device then print `Gdx.audio.getAvailableOutputDevices()` and it have a new item corresponding to my device.
Same for removing one. Calling `Gdx.audio.getAvailableOutputDevices()` again return a new Array without it.
I've triied to do the same with wire device. Same result.

With Windows 11, Java 18.
